### PR TITLE
Fix tag and prompt in attdef entity

### PR DESCRIPTION
**Pull request by michal-josef-spacek**

### CODE REVIEW

**Review by rurban — CHANGES_REQUESTED** [review]
This needs a special case for DXF

**Comment by michal-josef-spacek** [comment]
> This needs a special case for DXF

I don't understand what you are talking.

I added https://github.com/LibreDWG/libredwg/pull/618/commits/78dd6cc614f69366950882d0a34cb0c25725c3a4
I don't understand what the previous situation means. Is it ok?

**Comment by rurban** [comment]
> This needs a special case for DXF

Nope

File: src/dwg.spec [hunk 1]
```diff
@@ -530,8 +530,6 @@ DWG_ENTITY (ATTDEF)
 
   SUBCLASS (AcDbAttributeDefinition);
   DXF {
-    FIELD_T (prompt, 3);
```
Comment (rurban) [inline src/dwg.spec:533]: This is wrong

**Review by rurban — CHANGES_REQUESTED** [review]

**Comment by rurban** [comment]
> > This needs a special case for DXF
> 
> I don't understand what you are talking.
> 
> I added [78dd6cc](https://github.com/LibreDWG/libredwg/commit/78dd6cc614f69366950882d0a34cb0c25725c3a4) I don't understand what the previous situation means. Is it ok?

78dd6cc is wrong.
I thought r11 dxf needs 2 before 3, but I was wrong. 3 is before 2 even with r11 dxf.

**Comment by rurban** [comment]
merged the first version

**Comment by michal-josef-spacek** [comment]
> > > This needs a special case for DXF
> > 
> > 
> > I don't understand what you are talking.
> > I added [78dd6cc](https://github.com/LibreDWG/libredwg/commit/78dd6cc614f69366950882d0a34cb0c25725c3a4) I don't understand what the previous situation means. Is it ok?
> 
> [78dd6cc](https://github.com/LibreDWG/libredwg/commit/78dd6cc614f69366950882d0a34cb0c25725c3a4) is wrong. I thought r11 dxf needs 2 before 3, but I was wrong. 3 is before 2 even with r11 dxf.

Why is this wrong?
I have DWG and DXF (AC1015) which has tag before prompt in DXF and DWG.

**Pull request closed without merging**